project1-sps/script.js

```diff
@@ -67,21 +57,17 @@ var main = function (input) {
   console.log(`The number of wins is ${win}`);
   console.log(`The number of draws is ${draw}`);
   console.log(`The number of losses is ${lose}`);
-   
-  return result;
-
 
+  return result;
 };
 
-var rollDice = function(){
+var rollDice = function () {
   var randomDecimal = Math.random() * 3;
   var randomInteger = Math.floor(randomDecimal);
   var diceNumber = randomInteger + 1;
   return diceNumber;
-}
-
-
+};
 
 // user inputs one of "scissors", "paper", or "stone"
 // the program internally randomly chooses scissors, paper, or stone
-//  the program outputs whether the user won, the program won, or it's a draw.
\ No newline at end of file
+// the program outputs whether the user won, the program won, or it's a draw.
```

Comment:
Good that you have pseudocode! Put it on top in the future and it can double as a short summary for others that read your code so they have an understanding of what you are writing before they actually read your code!